File: content/code/fahrenheit_to_celsius_docstring.f90
!> @brief Convert Fahrenheit to Celsius
!! @param temp_f Temperature in Fahrenheit
!! @return Temperature in Celsius
function fahrenheit_to_celsius(temp_f) result(temp_c)
    implicit none
    real temp_f
    real temp_c
    temp_c = (temp_f - 32.0) * (5.0/9.0)
end function

File: content/code/fahrenheit_to_celsius_docstring.cpp
// @brief: Converts a temperature in Fahrenheit to Celsius
//
// @param: temp_f: Temperature in Fahrenheit
//
// @return: Temperature in Celsius
double fahrenheit_to_celsius(double temp_f) {
    auto temp_c = (temp_f - 32.0) * (5.0 / 9.0);
    return temp_c;
}
